datasets/bomen/kapenherplant/v1.0.0.json

```diff
@@ -0,0 +1,440 @@
+			},
+		 "ecoscanUitgevoerd": {
+				"type": "string",		  
+				"description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "einddatumTechnischOnderzoek": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "groeiplaatsDatumToezicht": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "groeiplaatsmaatregelDatumUitgevoerd": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "herplantGewenst": {
+				"type": "string",		  
+				"description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		"idUitOudebeheerindeling": {
+			  "type": "string",
+			  "description": "Uniek kenmerk van een beheerobject uit een oude beheerindeling (guid van geovisia).",
+			  "auth": "FP/MDW",
+			  "reasonsNonPublic": [
+				"5.2 1: Bevat persoonlijke beleidsopvattingen (bevat geen milieu-informatie)"
+			  ]
+			},
+		 "inspectiedatum": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld"
+			},
+		 "kapDatumToezicht": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "kaplijstBomen": {
+				"type": "string",		  
+				"description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "kapmaatregelDatumUitgevoerd": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "mutatiedatum": {
+			    "type": "string",
+			    "format": "date",
+			    "description": "Wordt later ingevuld",
+				"auth": "FP/MDW",
+				"reasonsNonPublic": [
+				  "5.1 2d: Zwaarwegend belang: inspectie, controle en toezicht door bestuursorganen"
+				]
+			},
+		 "otlAanleveringDatumAangeleverd": {
```

Comment:
Geen afkortingen in veldnamen, gebruik eventueel shortname attribuut

Reply:
gewijzigd (zie commit)